### ISSUE TITLE
make Autotools run ./autogen.sh?

### ISSUE THREAD

**Issue by porterjamesj** [JuliaLang/BinDeps.jl#82]
Is there a way to make the Autotools provider run autogen.sh before running configure? I'm trying to wrap a library that builds this way and having some trouble.

**Comment by jiahao** [comment]
bump

**Comment by tkelman** [comment]
Ask upstream to put up a tarball from the result of `make dist`? Or do it manually with a `BuildProcess`, but that would force all users to have autoconf, automake, etc installed.

**Comment by porterjamesj** [comment]
yeah I ended up hosting a tarball myself on my personal site; I've always meant to bring it up with upstream but never got around to it :/
